TYPO3.Neos/Classes/TYPO3/Neos/Domain/Strategy/AssetUsageInNodePropertiesStrategy.php

```diff
@@ -89,23 +90,16 @@ public function getUsageReferences(AssetInterface $asset)
             return $this->firstlevelCache[$assetIdentifier];
         }
 
-        $userWorkspace = $this->userService->getPersonalWorkspace();
-
         $relatedNodes = [];
         foreach ($this->getRelatedNodes($asset) as $relatedNodeData) {
-            $accessible = $this->domainUserService->currentUserCanReadWorkspace($relatedNodeData->getWorkspace());
-            if ($accessible) {
-                $context = $this->createContextMatchingNodeData($relatedNodeData);
-            } else {
-                $context = $this->createContentContext($userWorkspace->getName());
-            }
-            $site = $context->getCurrentSite();
+            $context = $this->createContextMatchingNodeData($relatedNodeData);
             $node = $this->nodeFactory->createFromNodeData($relatedNodeData, $context);
             $flowQuery = new FlowQuery([$node]);
-            /** @var \TYPO3\TYPO3CR\Domain\Model\NodeInterface $documentNode */
+            /** @var NodeInterface $documentNode */
             $documentNode = $flowQuery->closest('[instanceof TYPO3.Neos:Document]')->get(0);
-
-            $relatedNodes[] = new AssetUsageInNodeProperties($asset, $site, $documentNode, $node, $accessible);
+            $site = $context->getCurrentSite();
+            $accessible = $this->domainUserService->currentUserCanReadWorkspace($relatedNodeData->getWorkspace();
```

Comment:
sure I agree to the change, what I meant was if just checking if the user can read the workspace, but not making sure it can read the node in that particular workspace is good enough for checking if it's accessible..

not entirely sure if a workspace can be accessible without the node itself being due to some parent workspace relations, but it seemed like that was the issue you had..